test/unit/spec/connect.js

```diff
@@ -86,9 +86,6 @@ describe('connect', () => {
       [{ video: { renderDimensions: null } }, 'whose .video.renderDimensions is null', TypeError, 'object'],
       [{ video: { renderDimensions: true } }, 'whose .video.renderDimensions is not an object', TypeError, 'object'],
       [{ video: { renderDimensions: ['foo'] } }, 'whose .video.renderDimensions is an Array', TypeError, 'object'],
-      [{ video: { renderDimensions: { dominantSpeaker: null } } }, 'whose .video.renderDimensions.dominantSpeaker is null', TypeError, 'object'],
```

Comment:
maybe it would be good to keep one test where renderdimentions does have `dominantSpeaker` and verify that it gets ignored.

Reply:
@makarandp0 I don't think that is necessary. I already verified that the previous unit tests with "dominantSpeaker" failed.